<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="underlay" stretch="3,1">
      <item>
       <layout class="QVBoxLayout" name="cameraLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </item>
        <item>
         <widget class="QComboBox" name="camerasComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="filtersLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="BilateralFilterButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
            <fontweight>Black</fontweight>
           </font>
          </property>
          <property name="text">
           <string>Bilateral</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="GaussFilterButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
            <fontweight>Black</fontweight>
           </font>
          </property>
          <property name="text">
           <string>Gauss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NoFilterButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
            <fontweight>Black</fontweight>
           </font>
          </property>
          <property name="text">
           <string>Original</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
